Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -925,3 +999,15 @@ bool vtkSlicerMarkupsWidgetRepresentation3D::GetNthControlPointViewVisibility(in
     }
   return false;
 }
+
+//----------------------------------------------------------------------
+void vtkSlicerMarkupsWidgetRepresentation3D::UpdateInteractionPipeline()
+{
+  Superclass::UpdateInteractionPipeline();
+  vtkMRMLViewNode* viewNode = vtkMRMLViewNode::SafeDownCast(this->ViewNode);
+  if (!viewNode)
+    {
+    this->InteractionPipeline->Actor->SetVisibility(false);
+    return;
+    }
```

Comment:
Should a call to `this->InteractionPipeline->Actor->SetVisibility(true);` be added below ? If not, may be sensible to add a comment

Reply:
The same thing applies to both this and the previous comment, but final visibility is controlled by the top level superclass in vtkSlicerMarkupsWidgetRepresentation.

Adding a comment and moving the superclass call to the end of the function to make the order more logical.